setup.py

```diff
@@ -94,9 +94,6 @@ def get_git_sha():
         'thrift-sasl>=0.2.1',
         'unicodecsv',
         'unidecode>=0.04.21',
-        'croniter==0.3.25',
```

Comment:
Are these deps not required anymore?

Reply:
(apologies for the late reply here. I was on vacation and traveling)

These dependencies are still needed, but they are specified in `requirements.txt` already. Did not want to keep it in two places.

Reply:
`requirements.txt` is a pip-compiled, "pinned" version of what's in here. We probably want to keep this in here though I think it belong in an `extra_requires` sub-category, but that's outside the scope of this PR. For now let's just keep this in the repo. Mind remove these edits from this PR?

Reply:
@mistercrunch removed the edits, but did alphabetically sort it. Let me know if you are OK with it and I will merge it in.